internal/saucecloud/cloud.go

```diff
@@ -618,10 +618,10 @@ var (
 	otherAppsUpload    uploadType = "other applications"
 )
 
-func (r *CloudRunner) uploadProjects(filename []string, pType uploadType, dryRun bool) ([]string, error) {
+func (r *CloudRunner) uploadProjects(filename, descriptions []string, pType uploadType, dryRun bool) ([]string, error) {
 	var IDs []string
-	for _, f := range filename {
-		ID, err := r.uploadProject(f, pType, dryRun)
+	for i, f := range filename {
```

Comment:
Related to comment above
```suggestion
	for i, f := range filenames {
```